app/src/main/java/practicum/getfitla_v3/Exercise.java

```diff
@@ -5,58 +5,94 @@
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
 import android.view.View;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.util.ArrayList;
 import java.util.List;
 import android.content.Intent;
-import android.content.Context;
+
 
 public class Exercise extends AppCompatActivity implements ItemClickListener{
 
+    List<ExerciseItemFormat> exerciseList = new ArrayList<>(3);
```

Comment:
Just a general note, you don't have to initialize the ArrayList with a size. Because they have dynamic memory, you can just do: 
```
List<ExerciseItemFormat> exerciseList = new ArrayList<~>();
```